packages/terra-worklist-data-grid/src/WorklistDataGrid.jsx

```diff
@@ -505,6 +624,7 @@ function WorklistDataGrid(props) {
         aria-label={ariaLabel}
         className={cx('worklist-data-grid', theme.className)}
         onKeyDown={handleKeyDown}
+        onKeyUp={handleKeyUp}
```

Comment:
I would not expect us to have to handle a key up event.  This makes me think there is an issue with the design.

Reply:
I still don't believe we should need a key up event.  We should not have manually track the Shift key.